docs/variants/protectli_v1x10/hardware-matrix.md

```diff
@@ -30,12 +30,12 @@ coreboot port on the Protectli V1210/V1410/V1610 firewall.
 |--------------------------------|---------------------------------------------|
 | **CPU**                        | Intel(R) Celeron(R) N5105 @ 2.00GHz         |
 | **RAM**                        | Micron MT53D512M64D4RQ-046                  |
-| **Flash memory**               | Winbond 25Q128JVSIQ                         |
+| **Flash memory**               | Winbond W25Q128.V                           |
```

Comment:
Leave the full chip name, `W25Q128.V` is just a flashrom chip name with a wildcard...